deploy/deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       serviceAccountName: calico-host-endpoint-controller
       containers:
-      - image: quay.io/kinvolk/calico-endpoint-controller:v0.0.2
+      - image: kinvolk/calico-endpoint-controller:v0.0.2
```

Comment:
Why this? It's not mentioned in commit message.

Reply:
The readme explains how to build the multi-arch image for docker hub. quay.io (as far as we experienced) does not support this.